photosynthesis_metrics/utils.py

```diff
@@ -31,10 +30,9 @@ def _validate_input(x: torch.Tensor, y: torch.Tensor, kernel_size: Optional[int]
     if kernel_size is not None:
         assert kernel_size % 2 == 1, f'Kernel size must be odd, got {kernel_size}.'
     if scale_weights is not None:
-        assert isinstance(scale_weights, (list, tuple)), \
+        assert isinstance(scale_weights, (list, tuple, torch.Tensor)), \
             f'Scale weights must be of type list or tuple, got {type(scale_weights)}.'
-        assert len(scale_weights) == 4, f'Scale weights collection must contain 4 values, got {len(scale_weights)}.'
-    
+        assert len(scale_weights) == 5, f'Scale weights collection must contain 5 values, got {len(scale_weights)}.'
```

Comment:
Could you comment on why it changed from 4 to 5 during the refactoring?

Reply:
Because `len((0.0448, 0.2856, 0.3001, 0.2363, 0.1333)) == 5`, it's weights from paper.  The reason this tests never failed before is that they were never called (scale_weights were equal to None)